mvcoverride.php

```diff
@@ -226,7 +226,7 @@ private function getOverrideFileInfo($includePath, $component, $filePath, $type
 			$baseName = basename($filePath);
 			$prefix = substr($baseName, 0, 5);
 
-			if (($app->isAdmin() && $prefix == 'admin') || (!$app->isAdmin() && $prefix != 'admin') )
+			if (($app->isAdmin() && $prefix == 'admin') || (!$app->isAdmin() && $prefix != 'admin') || !(!$app->isAdmin() && $prefix == 'admin'))
```

Comment:
Right rule, what i try add - if i create helper for using in admin side, but original file in frontend side(frontend helper files not have in name `admin` prefix) - you example return false.